src/components/ProjectDisciplines.js

```diff
@@ -37,7 +44,8 @@ class ProjectDisciplines extends React.Component {
           title={label}
           tag={value}
           key={idx}
-          color={color} /> )
+          color={color}
+          setSelectedProjectTag={() => {this.props.setSelectedProjectTag(value)}} /> )
```

Comment:
Should this prop `setSelectedProjectTag` be defined in the `Discipline` component? I'm not seeing any reference to it there, but perhaps I'm missing something.